elide-spring/elide-spring-boot-autoconfigure/src/test/resources/models/tables/playerStats.hjson

```diff
@@ -0,0 +1,56 @@
+{
+    tables: [{
+        name: PlayerStats
+        table: PlayerStats
+        description:
+        '''
+        A long description
+        '''
+        cardinality : large
+        readAccess : Prefab.Role.All
+        joins: [
+            {
+                name: playerCountry
+                to: PlayerCountry
+                type: toOne
+                definition: '%join.id = %from.countryId'
+            }
+        ]
+        measures : [
+            {
+            name : highScore
+            type : INTEGER
+            definition: 'MAX(highScore)'
+            readAccess : Prefab.Role.All
+            }
+        ]
+        dimensions : [
+            {
+                name : name
+                type : TEXT
+                readAccess : Prefab.Role.All
```

Comment:
SInce this is defined at the table level, you don't need to define it for each field.